website/src/components/pricing/PricingPlan.tsx

```diff
@@ -42,12 +43,12 @@ const FEATURE_INFO: Record<keyof Features, FeatureInfo> = {
         description: 'Code navigation for 30+ languages, with hovers, definitions, and references across repositories',
     },
     batchChanges: {
-        label: 'Batch changes',
+        label: 'Batch changes (available add-on)',
```

Comment:
@jeanduplessis - is it possible to make this all on one line? Right now, "on" is on its own line.

Reply:
Additionally, can Batch Changes be capitalized?